distributed-job-manager/src/main/java/ru/fix/distributed/job/manager/model/AssignmentState.java

```diff
@@ -0,0 +1,158 @@
+package ru.fix.distributed.job.manager.model;
+
+import java.util.*;
+
+/**
+ * ZookeeperState represent Map with mapping workers to  work items
+ * and provide additional methods for easier Zookeeper state reconstruction
+ */
+public class AssignmentState extends HashMap<WorkerItem, List<WorkItem>> {
+
```

Comment:
Looks like you disliked scheme 
AssignmentState extends HashMap<JobId, HashMap<WorkerId, HashSet<WorkItem>>>

well, at least let's make set of WorkItem not a list: 

AssignmentState extends HashMap<WorkerId, HashSet<WorkItem>> 
?

Reply:
There is uncomfortable to work (read and write zk tree) with scheme AssignmentState extends HashMap<JobId, HashMap<WorkerId, HashSet>>

Reply:
ok,then at least  let's make 
AssignmentState extends HashMap<WorkerId, **HashSet**<WorkItem>>
?